internal/dispatch/graph/lookupresources2_test.go

```diff
@@ -701,6 +702,10 @@ func TestLookupResources2OverSchemaWithCursors(t *testing.T) {
 					foundChunks := [][]*v1.DispatchLookupResources2Response{}
 					for {
 						stream := dispatch.NewCollectingDispatchStream[*v1.DispatchLookupResources2Response](ctx)
+
+						uintPageSize, err := safecast.ToUint32(pageSize)
+						require.NoError(err)
```

Comment:
I made a bunch of these changes to tests. They're overkill, especially since the values are statically defined in most cases, so I can walk these back out if desired.